Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/Controller.purs

```diff
@@ -443,3 +488,92 @@ type ServiceabilityResponse = {
+    void $ launchAff $ flowRunner defaultGlobalState $ do 
+      let newRoute = case state.data.route of 
+                        MB.Just (Route route) ->
+                          let (Snapped routePts) = route.points 
+                              newPts = if DA.length routePts > 1 then 
+                                        getExtendedPath $ walkCoordinates (route.points)
+                                        else 
+                                          walkCoordinate startLat startLon endLat endLon
+                              newRoute = route {points = Snapped (map (\item -> LatLong { lat : item.lat, lon : item.lng}) newPts.points)}
+                          in (newRoute)
+                        MB.Nothing -> 
+                          let endLatitude = if endLat == 0.0 then startLat else endLat 
+                              endLongitude = if endLon == 0.0 then startLon else endLon
+                              newPts =walkCoordinate startLat startLon endLatitude endLongitude
+                              newRoute = {boundingBox: MB.Nothing, distance: 0, duration: 0, pointsForRentals: MB.Nothing, points : Snapped (map (\item -> LatLong { lat : item.lat, lon : item.lng}) newPts.points), snappedWaypoints : Snapped []}
+                          in newRoute
+      liftFlow $ drawRoute [ (walkCoordinates newRoute.points)] "LineString" true sourceMarkerConfig destMarkerConfig 8 "NORMAL" (specialLocationConfig "" "" false getPolylineAnimationConfig) (getNewIDWithTag "SearchLocationScreenMap")
+    pure NoAction
+  ] 
+
+quotesFlow res state = do
+  let quoteList = getQuotesTransformer res.quotes state.appConfig.estimateAndQuoteConfig
+      filteredQuoteList = (getFilteredQuotes res.quotes state.appConfig.estimateAndQuoteConfig)
+      sortedByFare = DA.sortBy compareByFare filteredQuoteList
+      rentalsQuoteList =  (DA.mapWithIndex (\index quote -> 
+    let quoteDetails = transformQuote quote index 
+        fareDetails = case quote of 
+          RentalQuotes body -> 
+            let (QuoteAPIEntity quoteEntity) = body.onRentalCab
+            in case (quoteEntity.quoteDetails)^._contents of
+              (RENTAL contents) -> 
+                let (RentalQuoteAPIDetails quoteDetails) = contents 
+                in (transFormQuoteDetails quoteDetails)
+              _ -> dummyFareQuoteDetails
+          _  -> dummyFareQuoteDetails
+    in { quoteDetails : quoteDetails, index : index, activeIndex : 0 , fareDetails : fareDetails}
+    ) sortedByFare)
+  continue state { data{quotesList = rentalsQuoteList, selectedQuote = if MB.isNothing state.data.selectedQuote then DA.head $ rentalsQuoteList else state.data.selectedQuote}}
+  where 
+    transFormQuoteDetails quoteDetails = 
+        { includedKmPerHr : MB.fromMaybe 0 quoteDetails.includedKmPerHr 
+        , nightShiftCharge : MB.fromMaybe 250 quoteDetails.nightShiftCharge 
```

Comment:
why 250?

Reply:
this is the default value that we are have been using